packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -3702,6 +3706,22 @@ export interface ReactionButtonStrings {
     tooltipDisabledContent?: string;
 }
 
+// @beta
+export type ReactionResourceParams = {
+    url: string;
+    frameCount: number;
+    size?: number;
+};
+
+// @beta
+export interface ReactionResources {
+    applaseReaction?: ReactionResourceParams;
+    heartReaction?: ReactionResourceParams;
+    laughReaction?: ReactionResourceParams;
+    likeReaction?: ReactionResourceParams;
+    surprisedReaction?: ReactionResourceParams;
```

Comment:
nit: surprised -> surprise (to match the rest that are in present tense)

Reply:
So, for some reason, in teams the reaction name is surprised and to keep it similar, I've used `surprised` everywhere.

Reply:
That would be changing the API already in GA and are we allowed to do that? 
For Web UI -> since we are still beta, we can update. Lets discuss separately and that update can go as a separate PR?

Reply:
Alright. Keeping them aligned. :)